### ISSUE TITLE
mainModulePath undefined. Package fails to activate

### ISSUE THREAD

**Issue by basarat** [TypeStrong/atom-typescript#162]
![image](https://cloud.githubusercontent.com/assets/874898/6536415/9a769b20-c4a1-11e4-9991-fcad4f72db2b.png)
Any body else noticed this? Our package fails to `activate` because of this. 

@park9140 might know something about this

**Comment by basarat** [comment]
removing `autocomplete-plus` and letting apd reinstall it fixes it. Investigating

**Comment by park9140** [comment]
@basarat, can you document what changes you made to apd that required including it in the project?

**Comment by park9140** [comment]
Just for future reference it would be better to fork apd then have us npm install a branch directly from github rather than merge it into our repo.

**Comment by basarat** [comment]
@park9140 will document it. Sorry about including it. I see the advantage of forking (better commits and diffs). 

Found the reason. I ran "update all" last night but before it completed I shut down atom (was too tired and assumed something in background would compete it). This results in an empty directory with just folders: 

![image](https://cloud.githubusercontent.com/assets/874898/6536538/74b57478-c4a2-11e4-9c53-4fe48af3f9cc.png)

Perhaps I should add a guard against this case but its no longer *high priority*.
